openslides/users/config_variables.py

```diff
@@ -11,12 +11,12 @@ def get_config_variables():
     # Sorting
     yield ConfigVariable(
         name='users_sort_by',
-        default_value='firstName',
+        default_value='first_name',
         input_type='choice',
         label='Sort name of participants by',
         choices=(
-            {'value': 'firstName', 'display_name': 'First name'},
-            {'value': 'lastName', 'display_name': 'Last name'}),
+            {'value': 'first_name', 'display_name': 'First name'},
+            {'value': 'last_name', 'display_name': 'Last name'}),
```

Comment:
If you change the config value you have to change also the get_short_name function in /users/base.js:
https://github.com/OpenSlides/OpenSlides/blob/30fd55409b1b5ab919a87afabddae126d867915f/openslides/users/static/js/users/base.js#L104